### PR TITLE
Add jupyter-black

### LINKED ISSUE #37: Add jupyter-black
https://github.com/n8henrie/jupyter-black

### PR DESCRIPTION
**Pull request by hicksl**
Remove the nb_black dependency from pyproject.toml and add jupyter-black (https://github.com/n8henrie/jupyter-black)

Closes #37.

### CODE REVIEW

**Comment by hicksl** [comment]
Passing manual checks.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ repos:
           - flake8-return==1.1.3
         args: ["--config=setup.cfg"]
   - repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.3.0
```
Comment (john-sandall) [inline .pre-commit-config.yaml:58]: This should be reflected in the inner precommit config

Comment (john-sandall) [inline .pre-commit-config.yaml:58]: This should be mirrored in the inner precommit config

**Review by john-sandall — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (john-sandall) [reply .pre-commit-config.yaml:58]: We can do this in a separate PR though

Comment (john-sandall) [reply .pre-commit-config.yaml:58]: #56

**Pull request merged**